gtk4/sys/Cargo.toml

```diff
@@ -64,6 +64,9 @@ version = "4.18"
 [package.metadata.system-deps.gtk4.v4_20]
 version = "4.19"
 
+[package.metadata.system-deps.gtk4.v4_19]
+version = "4.19"
```

Comment:
IIUC we'd want to use the 4.20 above

Reply:
https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/8819